src/foam/nanos/crunch/ui/CapableWAO.js

```diff
@@ -94,7 +80,7 @@ foam.CLASS({
 
       // Set transient 'capability' property if it exists
       var prop = wizardlet.of.getAxiomByName('capability');
-      if ( prop ) prop.set(loadedData, this.capability);
+      if ( prop ) prop.set(loadedData, wizardlet.capability);
```

Comment:
Not all wizardlets have capabilities, this wont work

Reply:
Hmm, i am not sure this the right problem to solve, either the saver should ignore wizardlets without associated capabilities or throw an error when using capable saver on non-capability wizardlets